src/Site_Command.php

```diff
@@ -66,13 +66,18 @@ public static function get_site_types() {
 	public function __invoke( $args, $assoc_args ) {
 
 		$site_types = self::get_site_types();
-		$assoc_args = $this->convert_old_args_to_new_args( $args, $assoc_args );
 
-		if ( isset( $assoc_args['type'] ) ) {
-			$type = $assoc_args['type'];
-			unset( $assoc_args['type'] );
+		// default site-type
+		$type = 'html';
+
+		if ( in_array( reset( $args ), [ 'create', 'update' ] ) || empty( $args ) ) {
+			$assoc_args = $this->convert_old_args_to_new_args( $args, $assoc_args );
```

Comment:
I think we discuss to add this logic in `convert_old_args_to_new_args` function since we are restricting support of old arg so this logic should be in `convert_old_args_to_new_args`.